### DISCUSSION TITLE
Regex bug with "save button"

### DISCUSSION THREAD

**Discussion by Soulage23** [Budibase/budibase#4683] [Help]
Hi there!

Trying to make validation, what expects to receive 10 or 12 digits.
![2022-02-24_17-29-01](https://user-images.githubusercontent.com/17461986/155543889-313c5c19-6779-465d-bd63-d1ddd556e5a4.png)

11 digits pass the validation. But regex101.com says that its not.
![image](https://user-images.githubusercontent.com/17461986/155544044-d3b96ead-02b7-4f89-ab6a-e07f2e135c04.png)

Im dumb or not?

**Comment by Soulage23** [comment]
Oh, may be its okay, just "save" button bug.
https://drive.google.com/uc?id=1azIR_ZW6YKIjEe0ZKJ6FzXQ1JkYI_JoM

**Reply by mjashanks** [reply]
Thanks for reporting!
